Kitodo/src/test/java/org/kitodo/production/services/data/ImportServiceIT.java

```diff
@@ -83,31 +91,73 @@ public static void cleanDatabase() throws Exception {
     @Test
     public void testImportProcess() throws Exception {
         Assert.assertEquals("Not the correct amount of processes found", 7, (long) processService.count());
-        File script = new File(ConfigCore.getParameter(ParameterCore.SCRIPT_CREATE_DIR_META));
-        if (!SystemUtils.IS_OS_WINDOWS) {
-            ExecutionPermission.setExecutePermission(script);
-        }
-        Process importedProcess = importService.importProcess(RECORD_ID, 1, 1,
-                MockDatabase.getK10PlusImportConfiguration(), new HashMap<>());
-        if (!SystemUtils.IS_OS_WINDOWS) {
-            ExecutionPermission.setNoExecutePermission(script);
-        }
-
+        Process importedProcess = importProcess(RECORD_ID, MockDatabase.getK10PlusImportConfiguration());
         Assert.assertEquals("WrongProcessTitle", "Kitodo_" + RECORD_ID, importedProcess.getTitle());
         Assert.assertEquals("Wrong project used", 1, (long) importedProcess.getProject().getId());
         Assert.assertEquals("Wrong template used", 1, (long) importedProcess.getTemplate().getId());
         Assert.assertEquals("Not the correct amount of processes found", 8, (long) processService.count());
     }
 
-    private static void setupServer(String serverResponse) {
+    @Test
+    public void shouldCreateUrlWithCustomParameters() throws DAOException, ImportException, IOException {
```

Comment:
The naming does not fully describe what is happeing in the test as the creation of url parameter is happening in the MockDatabase class and not in this test. Maybe this should be renamed to reflect what is happening in the test (`shouldCreateProcessWithCustomUrlParameters.`)  and also check if the imported processes is the expected one and not just "not null"

Reply:
Actually, the MockDatabase only prepares the ImportConfiguration with custom URL parameters. This test does not check the creation of the _parameters_, though, but instead the creation of the _URL_ itself, using the parameters of the ImportConfiguration. For that reason I think the method is named correctly. 

The creation of an ImportConfiguration with custom URL parameters on the other hand is tested in the Selenium Test `AddingST.addCustomImportconfigurationWithUrlParameters`.